event_project/models/event_event.py

```diff
@@ -0,0 +1,105 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright :
+#        (c) 2014 Antiun Ingenieria, SL (Madrid, Spain, http://www.antiun.com)
+#                 Endika Iglesias <endikaig@antiun.com>
+#                 Antonio Espinosa <antonioea@antiun.com>
+#                 Javier Iniesta <javieria@antiun.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+from datetime import datetime, timedelta
+
+
+class EventEvent(models.Model):
+    _inherit = 'event.event'
+
+    project_template = fields.Many2one(
+        comodel_name='project.project', string='Template project',
+        domain="[('state', '=', 'template')]")
+    project_related = fields.Many2one(
+        comodel_name='project.project', string='Related project',
+        readonly=True)
+
+    def reorganize_project(self, project, event, date_begin=None):
```

Comment:
For betetr separation of concerns, shouldn't this logic be on project.project, and this method only make a call on it?